Makefile

```diff
@@ -60,72 +60,40 @@ override LIBC_TOP_HALF_MUSL_DIR = $(LIBC_TOP_HALF_DIR)/musl
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/misc/nftw.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/errno/strerror.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/network/htonl.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/network/htons.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/network/ntohl.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/network/ntohs.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/network/inet_ntop.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/network/inet_pton.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/network/inet_aton.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/network/in6addr_any.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/network/in6addr_loopback.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/fenv/fenv.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/fenv/fesetround.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/fenv/feupdateenv.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/fenv/fesetexceptflag.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/fenv/fegetexceptflag.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/fenv/feholdexcept.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/exit/exit.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/exit/atexit.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/exit/assert.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/exit/quick_exit.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/exit/at_quick_exit.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/time/strftime.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/time/asctime.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/time/asctime_r.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/time/ctime.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/time/ctime_r.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/time/wcsftime.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/time/strptime.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/time/difftime.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/time/timegm.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/time/ftime.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/time/gmtime.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/time/gmtime_r.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/time/timespec_get.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/time/getdate.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/time/localtime.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/time/localtime_r.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/time/mktime.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/time/__tm_to_secs.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/time/__month_to_secs.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/time/__secs_to_tm.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/time/__year_to_secs.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/time/__tz.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/fcntl/creat.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/dirent/alphasort.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/dirent/versionsort.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/env/clearenv.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/env/getenv.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/env/putenv.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/env/setenv.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/env/unsetenv.c \
-    $(LIBC_TOP_HALF_MUSL_SRC_DIR)/unistd/posix_close.c \
+    $(addprefix $(LIBC_TOP_HALF_MUSL_SRC_DIR)/, \
+        misc/a64l.c \
+        misc/basename.c misc/dirname.c \
```

Comment:
What rule have use used to join these file onto the same line?

Any reason not to stick with one-line-per-file?  (I normally like to do this to make diff in particular more readable)

Reply:
It's just ad-hoc groupings of things that are related. I find reducing the number of lines makes a file easier to read.

Reply:
But I don't feel strongly about it. I've now submitted a change to put each file on its own line.

Reply:
But, I don't feel strongly about it, so I've now changed it to put each file on its own line.